snap/snapcraft.yaml

```diff
@@ -0,0 +1,118 @@
+name: edgex-device-onvif-camera
+base: core20
+license: Apache-2.0
+adopt-info: metadata
+
+architectures:
+  - build-on: amd64
+  - build-on: arm64
+
+grade: stable
+confinement: strict
+
+slots:
+  edgex-secretstore-token:
+    interface: content
+    source:
+      write: [$SNAP_DATA/device-onvif-camera]
+
+plugs:
+  device-config:
+    interface: content
+    target: $SNAP_DATA/config/device-onvif-camera
+
+apps:
+  device-onvif-camera:
+    adapter: full
+    command: bin/device-onvif-camera -confdir $SNAP_DATA/config/device-onvif-camera -profile res --registry $CONSUL_ADDR
+    command-chain:
+      - bin/source-env-file.sh
+    environment:
+      CONSUL_ADDR: "consul://localhost:8500"
+      APPCUSTOM_PROVISIONWATCHERDIR: $SNAP_DATA/config/device-onvif-camera/res/provision_watchers
+      DEVICE_PROFILESDIR: $SNAP_DATA/config/device-onvif-camera/res/profiles
+      DEVICE_DEVICESDIR: $SNAP_DATA/config/device-onvif-camera/res/devices
+      SECRETSTORE_SECRETSFILE: $SNAP_DATA/device-onvif-camera/onvif-credentials.json
+      SECRETSTORE_DISABLESCRUBSECRETSFILE: "false"
```

Comment:
Since this is missing from the configuration.toml file, does it have any effect when this env var is set?
It should already default to false.